build_tools/build_source.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Build the package
+# Install the environment
+chmod u+x ./build_tools/env.sh
+./build_tools/env.sh
+
+# Build the package
+mkdir build
+cd build
+cmake .. && make pyrfr_docstrings
+cd python_package
+
+# Copy required files, which will be
+# available to a docker image that
```

Comment:
Just checking, the mention of a docker container here is a copy paste error?

Reply:
Actually, a docker image is internally downloaded and used to create the manylinux images. That is what is meant here

Reply:
But this comment is inside the `build_source.sh` and not the `build_wheel.sh`. Could you briefly explain why we need a manylinux image for the source distribution?

Reply:
Sorry, I missed this file. I checked the wrong one. None of this is necessary anymore, I have removed this.